packages/doenetml-iframe/vite.config.ts

```diff
@@ -0,0 +1,33 @@
+import { defineConfig } from "vite";
+import dts from "vite-plugin-dts";
+
+// These are the dependencies that will not be bundled into the library.
+const EXTERNAL_DEPS = ["react", "react-dom"];
+
+// https://vitejs.dev/config/
+export default defineConfig({
+    base: "./",
+    plugins: [dts({ rollupTypes: true })],
+    build: {
+        minify: false,
```

Comment:
Is there a reason we aren't minifying these (not they are at all big)?

Reply:
Minifying should only be done at the final stage for end-user code that is deployed. Since this is to be included as a library in other code, the calling code is responsible for minifying. There's no point in minifying twice (and I think it's kinda mean, in case someone has to debug something).

All that is to say, `standalone` (and maybe the worker) is the only package that currently comes to mind that I think should be minified.